lib/formatters/stringFormatter.js

```diff
@@ -200,6 +203,8 @@ function formatter(messages, source, cwd) {
 		return row;
 	});
 
+	const hasNonAsciiChar = messages.some((msg) => NON_ASCII_OR_WS_PATTERN.test(msg.text));
```

Comment:
[suggest] I'm unsure why, but it seems we don't need to call `getMessageWidth()` twice. Could you please reduce evaluation times?

```suggestion
	const messageWidth = getMessageWidth(columnWidths);
	const hasNonAsciiChar = messages.some((msg) => NON_ASCII_OR_WS_PATTERN.test(msg.text));
```